lib/token_authentication.rb

```diff
@@ -2,6 +2,7 @@ class TokenAuthentication < Faraday::Middleware
   def call(env)
     if RequestStore.store[:auth_token]
       env[:request_headers]["Authorization"] = "Bearer #{RequestStore.store[:auth_token]}"
+      env[:request_headers]["HTTP_USER_AGENT"] = "#{'(development-env) ' if Rails.env.development?}SAGN Customers Web app Revision-#{`git rev-parse --short HEAD`.strip}"
```

Comment:
@deepali1-aconnexion let's maybe place (development) in the end of User Agent?

Like

```
"SAGN Customers Web app Revision-#{`git rev-parse --short HEAD`.strip} (#{Rails.env})"
```